test/unit/spec/util/jobLogViewerSpec.js

```diff
@@ -424,7 +425,7 @@ define([
              * job state object is received.
              */
             JobsData.example.JobState.invalid.forEach((state) => {
-                it(`should not do anything when given dodgy job state ${JSON.stringify(
+                xit(`should not do anything when given dodgy job state ${JSON.stringify(
```

Comment:
not sure this is worth keeping as the job comm channel should be validating all job messages prior to sending them out